js/src/common/Component.ts

```diff
@@ -33,7 +29,7 @@ export type ComponentAttrs = {
  *
  * @see https://mithril.js.org/components.html
  */
-export default abstract class Component<T extends ComponentAttrs = any> implements Mithril.ClassComponent<T> {
+export default abstract class Component<T extends ComponentAttrs> implements Mithril.ClassComponent<T> {
```

Comment:
Why remove `= any`? To be honest, it *should* be `= ComponentAttrs` instead, so that TS linters (IDEs, etc) do not complain when a component doesn't pass its own interface.

Reply:
I thought it wasn't necessary tbh. But yeah that's a good point, I'll add that back in